config/routes.rb

```diff
@@ -11,6 +11,7 @@
    namespace :obituaries do
      get 'covid-19', to: 'covid#index'
      get 'recent', to: 'recent#index'
+     get 'community', to: 'community#index'
```

Comment:
I have to say, it is nice seeing the word "community" after looking at the word "obituary" so much. Just a tad more uplifting :)